website/content/docs/release-notes/v0_18_0.mdx

```diff
@@ -228,5 +240,24 @@ description: >-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.18.x
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Boundary version 0.18.x controllers or workers are unable to establish SSH connections using the <code>boundary connect ssh</code> command
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Boundary version 0.18.x uses Go version 1.23, which introduced a new x509 key pair behavior. Some VPN implementations struggle with the TLS handshake being sent over 2 frames instead of 1, which can lead to Boundary version 0.18.x controllers or workers being unable to establish SSH connections.
+    <br /><br />
+    As a workaround, you can revert back to the previous key pair behavior by adding the <code>tlskyber=0</code> and <code>x509keypairleaf=0</code> parameters to the GODEBUG environment variable before the <code>boundary connect ssh command</code>. For example:
+    <br /><br />
+    <code>GODEBUG=tlskyber=0,x509keypairleaf=0 boundary connect ssh -target-id&lt;ID&gt;</code>
+    <br /><br />
+    Learn more: <a href="https://tip.golang.org/doc/go1.23">Go 1.23 Release Notes</a>
```

Comment:
I think we should add a link to the issue where this is discussed, which will help users figure out that actually this is a VPN problem, not a Boundary problem: https://github.com/golang/go/issues/70047

Reply:
Sounds good! I added the link to the issue in addition to the release notes, but I can also remove the release notes link if you don't think they're relevant.